www/html/ci/application/views/base.html

```diff
@@ -4,72 +4,76 @@
-                            <li class="navbar-tab"><a href="#smash">Random</a></li>
-                            <li class="dropdown"><a aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="#" role="button">Tutorials</a>
-                                <ul class="dropdown-menu">
-                                    <li class="navbar-tab"><a href="#physics">Physics</a></li>
-                                    <li class="navbar-tab"><a href="#math">Math</a></li>
-                                </ul>
-                            </li>
-                            <li class="navbar-tab"><a href="#contactme">Contact</a></li>
-                            {% if true == session %}
-                            <li class="navbar-tab"><a href="#adminpage">Admin</a></li>
-                            {% endif %}
 
-                        </ul>
+<div class="container kt-tabs-component">
+<div class="container strictly-no-padding ">
+<div class="kt-google-translate-widget" id="google_translate_element"></div>
+<nav class="navbar navbar-inverse kt-tabs-component-nav"><!--navbar-default-->
+  <div class="container-fluid">
+    <div class="navbar-header">
 
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
 
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
 
-
-                        </ul>
-
-
+      <a class="navbar-brand" href="#" style="color:#fed136;">Menu</a>
+    </div>
+    <div class="collapse navbar-collapse kt-tabs-component-nav-div" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#aboutsite">About This Site</a></li>
+        <li><a href="#whatsup">What's Up?</a></li>
+		<li><a href="#aboutme">About Me</a></li>
+		<li><a href="#gamedev">Games</a></li>
+		<li><a href="#otherprojects">Other Projects</a></li>
+		<li><a href="#smash">Random</a></li>
+		<li class="dropdown"><a aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="#" role="button">Tutorials</a>
+			<ul class="dropdown-menu">
+				<li><a href="#physics">Physics</a></li>
+				<li><a href="#math">Math</a></li>
+			</ul>
+		</li>
+		<li><a href="#contactme">Contact</a></li>
+    {% if true == session %}
+        <li><a href="#adminpage">Admin</a></li>
+    {% endif %}
+
+
+
+      </ul>
+</ul>
                     </div>
                 </div>
             </nav>
-
         </div>
     </div>
-    {% block content %}{% endblock %}
+    {% block content1 %}{% endblock %}
+
+
+    	<footer class="container-fluid center-block">
+    		<table><tr>
+    			<td><h6 id="copyright">Copyright KenTorii.com</h6></td>
+          {% if true == session %}
+    			<td><a type="button" data-toggle="modal" data-target="#myModal"><i class="fa fa-2x fa-unlock" onmouseover="lock(this)" onmouseout="unlock(this)" id="lock" aria-hidden="true"></i></a></td>
```

Comment:
If I'm logged in, and then I click on the icon, then it should log me out immediately. Does your code do this? If not, please add it.